sigstore/_utils.py

```diff
@@ -32,6 +32,8 @@
 else:
     from importlib import resources
 
+import newtype
```

Comment:
Nonexistent module? I see you added `newtypes.py` so this should probably be `import newtypes` (but I'll leave a comment w/ thoughts about avoiding a new module).